src/components/Home/index.js

```diff
@@ -0,0 +1,19 @@
+import React, { Component } from "react";
+import { Jumbotron } from "react-bootstrap";
+import { Link } from 'react-router-dom';
+
+class Home extends Component {
```

Comment:
Components that do not have state can be converted to stateless functional components. [Detailed explanation here](https://tylermcginnis.com/functional-components-vs-stateless-functional-components-vs-stateless-components/)